store/cachekv/memiterator.go

```diff
@@ -51,6 +54,10 @@ func (mi *memIterator) Value() []byte {
 	if _, ok := mi.deleted[string(key)]; ok && !reCallingOnOldLastKey {
 		return nil
 	}
+
+	value := mi.Iterator.Value()
+	mi.eventManager.EmitResourceAccessReadEvent("iterator", key, value)
```

Comment:
Not for the validation but I found it to be really useful for debugging missing Access Ops as most of the time the keys are not in a readable format. 

We're also not committing the events returned from this store so I thought it would be fine to leave it here